lib/elixir/src/elixir.hrl

```diff
@@ -1,7 +1,7 @@
 -define(m(M, K), maps:get(K, M)).
 -define(ann(Opts), elixir_utils:get_ann(Opts)).
 -define(line(Opts), elixir_utils:get_line(Opts)).
--define(generated, [{generated, true}, {location, 0}]).
+-define(generated(Opts), elixir_utils:get_generated(Opts)).
```

Comment:
Right, but those are more useful since you have to frequently get anns/the line from the environment. This is only used in a couple of places? In any event not super important, just seemed like an opportunity to delete code. :)